Packages/Tracking/Core/Runtime/Scripts/LeapXRServiceProvider.cs

```diff
@@ -525,26 +497,14 @@ protected override long CalculateInterpolationTime(bool endOfFrame = false)
                 return 0;
             }
 
-#if SVR
-            if (_xr2TimewarpMode == TimewarpMode.Experimental_XR2)
-            {
-                return GetPredictedDisplayTime_LeapTime();
-            }
-            else
-            {
-                return _leapController.Now() - 16000;
-            }
-
-#elif UNITY_ANDROID
+#if UNITY_ANDROID
             return _leapController.Now() - 16000;
 #else
 
             return _leapController.Now()
-                    - (long)_smoothedTrackingLatency.value
```

Comment:
@MattGrayUL do you remember why line this was deleted? It seems like one that we wanna keep? (It could have been deleted at the same time that the `smoothedTrackingLatency`  in Awake in LeapServiceProvider)

Reply:
that's a mistake, we were going to remove both _smoothedTrackingLatency and _fixedOffset as we marked them as obsolete, but later discovered they were necessary. This line should be reverted too